ignored-tests.json

```diff
@@ -0,0 +1,179 @@
+  "Coverage specs CSSCoverage should report sourceURLs",
+  "Coverage specs CSSCoverage should report stylesheets that have no coverage",
+  "Coverage specs CSSCoverage should work with a recently loaded stylesheet",
+  "Coverage specs CSSCoverage should work with complicated usecases",
+  "Coverage specs CSSCoverage should work with empty stylesheets",
+  "Coverage specs CSSCoverage should work with media queries",
+  "Coverage specs CSSCoverage should work",
+  "Coverage specs JSCoverage includeRawScriptCoverage should include rawScriptCoverage field when enabled",
+  "Coverage specs JSCoverage includeRawScriptCoverage should not include rawScriptCoverage field when disabled",
+  "Coverage specs JSCoverage resetOnNavigation should NOT report scripts across navigations when enabled",
+  "Coverage specs JSCoverage resetOnNavigation should report scripts across navigations when disabled",
+  "Coverage specs JSCoverage should ignore eval() scripts by default",
+  "Coverage specs JSCoverage should ignore pptr internal scripts if reportAnonymousScripts is true",
+  "Coverage specs JSCoverage should not hang when there is a debugger statement",
+  "Coverage specs JSCoverage should not hang when there is a debugger statement",
+  "Coverage specs JSCoverage should not ignore eval() scripts if reportAnonymousScripts is true",
+  "Coverage specs JSCoverage should report multiple scripts",
+  "Coverage specs JSCoverage should report right ranges for \"per function\" scope",
+  "Coverage specs JSCoverage should report right ranges",
+  "Coverage specs JSCoverage should report scripts that have no coverage",
+  "Coverage specs JSCoverage should report sourceURLs",
+  "Coverage specs JSCoverage should work with conditionals",
+  "Coverage specs JSCoverage should work",
+  "device request prompt does not crash",
+  "Drag n' Drop should drag and drop",
+  "Drag n' Drop should drop using mouse",
+  "Drag n' Drop should drop",
+  "Emulate idle state changing idle state emulation causes change of the IdleDetector state",
+  "Emulation Page.emulate should support clicking",
+  "Emulation Page.emulate should work",
+  "Emulation Page.emulateCPUThrottling should change the CPU throttling rate successfully",
+  "Emulation Page.emulateMediaFeatures should work",
+  "Emulation Page.emulateMediaType should work",
+  "Emulation Page.emulateNetworkConditions should change navigator.connection.effectiveType",
+  "Emulation Page.emulateTimezone should throw for invalid timezone IDs",
+  "Emulation Page.emulateTimezone should work",
+  "Emulation Page.emulateVisionDeficiency should work",
+  "Emulation Page.viewport should be detectable by Modernizr",
+  "Emulation Page.viewport should detect touch when applying viewport with touches",
+  "Emulation Page.viewport should support landscape emulation",
+  "Emulation Page.viewport should support touch emulation",
+  "Evaluation specs Page.evaluate should fail for circular object",
```

Comment:
This one should not fail with BiDi given that we can use `internalID` to explicitly not run into circular issues. So this is a failure on Puppeteers side as it looks like.

Reply:
Yes, it's marked as expected to FAIL for BiDi so it is safe to ignore it here. This is still the expected behavior for CDP though.

Reply:
I have experiment with having the circular objects for BiDi I think once we have that we will flip the test and it will pass for BiDi fail for CDP, like we have previously done where BiDi adds more functionality.